examples/managed-identity/kubernetes.json

```diff
@@ -0,0 +1,38 @@
+{
+  "apiVersion": "vlabs",
+  "properties": {
+    "orchestratorProfile": {
+      "orchestratorType": "Kubernetes",
+      "kubernetesConfig": {
+        "useManagedIdentity": true

```

Comment:
Makes sense. I was thinking something along the lines of to make it extensible:

```
 "kubernetesConfig": {
     "identityType": "Managed"
   }
```

OR

```
 "kubernetesConfig": {
     "identity": {
          "type" : "Managed",
          "masterIdentity": "/.../Microsoft.Identity/master-sp",
        "nodeIdentity": "/.../Microsoft.Identity/node-sp",

     }
   }

```